book/website/collaboration/ppie-events/ppie-events-location.md

```diff
@@ -0,0 +1,74 @@
+(cl-ppie-events-location)=
+# Location
+This section provides guidance that is most relevant for in-person or hybrid events. 
+
+The location of your event plays a pivotal role in supporting or hindering how accessible and inclusive your event will be for people with lived experience. As with all planning, reflect on how you will get input and feedback from your attendees with lived experience in order to identify the best venue for your event.
+
+
+(cl-ppie-events-choosing-venue)=
+## Choosing a Venue
+Before deciding on a venue it is recommended to have an organizer or member of the team visit potential venues to do a walk-through and check that all rooms in the venue will work for your event’s needs. 
+Include people with lived experience in this process as they can have expertise in identifying barriers such as inaccessibility for wheelchair users, poor lighting, or sensory overstimulation. 
+There is more guidance on how to choose an Accessible location in the Accessible Events Guide. 
+This can be helpful to both check the maintenance and layout of the space, and can also help you plan your event set-up such as where to put signs directing attendees to your event.
+
+Through a Universal Design lens, a venue that is able to support a diverse attendee list will be best for any event that involves both a research and general public audience. 
```

Comment:
```suggestion
Through a [Universal Design](https://universaldesign.ie/about-universal-design) lens, a venue that is able to support a diverse attendee list will be best for any event that involves both a research and general public audience. 
```

Reply:
should we link to an external website - or the section which discusses universal design in the chapter?